src/jit/importer.cpp

```diff
@@ -3714,6 +3714,34 @@ GenTreePtr Compiler::impIntrinsic(GenTreePtr            newobjThis,
             break;
         }
 
+        case CORINFO_INTRINSIC_GetRawHandle:
+        {
+            assert(IsTargetAbi(CORINFO_CORERT_ABI)); // Only CoreRT supports it.
+            CORINFO_RESOLVED_TOKEN resolvedToken;
+            resolvedToken.hMethod      = method;
```

Comment:
@MichalStrehovsky Is it necessary to set hMethod here? It is usually an output argument for resolvedToken.